### DISCUSSION TITLE
Define the landing page

### DISCUSSION THREAD

**Discussion by Andras-Csanyi** [dotnet/docfx#10261] [Q&A]
Hi,

Is there a way to define the landing page like the following?

I want docfx's visitors landing on the [Api](https://dotnet.github.io/docfx/api/Docfx.html) section and not in the main content. As of now they see the main content [here](https://dotnet.github.io/docfx/index.html).

**Comment by sliekens** [comment]
The best way I found is to add a file `index.md` with nothing except redirect like:

```yml
---
redirect_url: api/Docfx.html
---
```
